Configuration/TypoScript/setup.typoscript

```diff
@@ -1,8 +1,16 @@
 plugin.tx_tea {
   view {
-    templateRootPath = {$plugin.tx_tea.view.templateRootPath}
-    partialRootPath = {$plugin.tx_tea.view.partialRootPath}
-    layoutRootPath = {$plugin.tx_tea.view.layoutRootPath}
+    templateRootPaths {
+      0 = EXT:tea/Resources/Private/Templates/
```

Comment:
```suggestion
      0 = EXT:tea/Resources/Private/Templates/
      10 = {$plugin.tx_tea.view.templateRootPath}
```
And use the templates from the constant to override the ones provided by the extension